frontend-public/webpack.parts.js

```diff
@@ -1,11 +1,11 @@
 const webpack = require("webpack");
 const autoprefixer = require("autoprefixer");
-const cssnano = require("cssnano")({ zindex: false });
```

Comment:
@FreshGyan , did you implement this `zindex: false`? It was causing errors with webpack plugins.

Reply:
Nope! It came with the boilerplate we were using, if its not making our css files sad. then im good

Reply:
I made this change in `/frontend`. It was blindly copied over to `/frontend-public`, but I see you've removed it from there as well?

It was done to address a bug where the loading bar went over-top of the toast message. This was caused by a bug in one of our libraries. Perhaps your upgrades have removed this bug? If not, please verify that you're not reintroducing the bug.

The original PR: https://github.com/bcgov/mds/pull/317

Reply:
Thanks Nathan, I'll confirm whether the behaviour has been addressed.

Reply:
I can't replicate this issue on my machine.

Reply:
Okay. So long as it's not happening, I'm happy. Marking as resolved.